src/Config.cs

```diff
@@ -36,6 +50,8 @@ public static void OnModSettingsApplied()
             {
                 if (fieldInfo.FieldType == typeof(bool))
                     fieldInfo.SetValue(null, MelonPreferences.GetEntryValue<bool>(Category, fieldInfo.Name));
+                if (fieldInfo.FieldType == typeof(string) && fieldInfo.Name != nameof(Category))
```

Comment:
Why the fieldInfo.Name != nameof(Category) check? Was this necessary for this stuff to work somehow? I don't really get it :x
Also, this can be an "else if" (not that it actually matters).

Reply:
I think this is a similar remnant from when I was attempting to handle twitch directly in this mod.  I'll remove this.

Reply:
Removed.